src/stories/containers/TransparencyReport/components/TransparencyActuals/TransparencyActuals.tsx

```diff
@@ -28,6 +28,16 @@ interface Props {
 export const TransparencyActuals = (props: Props) => {
   const { isLight } = useThemeContext();
   const isMobile = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const {
+    checkOut,
+    handleChangeItemAccordion,
+    handleCheckedExpandedAll,
+    handleCloseModal,
+
+    headCountCategories,
+    noHeadCountCategories,
+    openModal,
+  } = useCategoriesContextModal();
```

Comment:
As before, this should no be needed once you move the hook to inside the modal

Reply:
done